src/GitVersion.Core/Core/MainlineBranchFinder.cs

```diff
@@ -22,25 +22,25 @@ internal class MainlineBranchFinder
         this.repositoryStore = repositoryStore.NotNull();
         this.repository = repository.NotNull();
         this.configuration = configuration.NotNull();
-        mainlineBranchConfigurations = configuration.Branches.Select(e => e.Value).Where(b => b.IsMainline == true).ToList();
+        mainlineBranchConfigurations = configuration.Branches.Select(e => e.Value).Where(b => b.IsMainBranch == true).ToList();
         this.log = log.NotNull();
     }
 
     public IDictionary<string, List<IBranch>> FindMainlineBranches(ICommit commit)
     {
         var branchOriginFinder = new BranchOriginFinder(commit, this.repositoryStore, this.configuration, this.log);
         return this.repository.Branches
-            .Where(BranchIsMainline)
+            .Where(BranchIsMainBranch)
             .Select(branchOriginFinder.BranchOrigin)
             .Where(bc => bc != BranchCommit.Empty)
             .GroupBy(bc => bc.Commit.Sha, bc => bc.Branch)
             .ToDictionary(group => group.Key, x => x.ToList());
     }
 
-    private bool BranchIsMainline(INamedReference branch)
+    private bool BranchIsMainBranch(INamedReference branch)
```

Comment:
No need to repeat the word `Branch` here, I think.

```suggestion
    private bool IsMainBranch(INamedReference branch)
```